db/seeds.rb

```diff
@@ -1,2 +1,8 @@
 Employer.create(name: 'Urtu', username: 'urtu', password: 'phoenix', company_name: 'Urtu Inc.')
-Employee.create(name: 'Ryan Kwon', username: 'rhk1', password: 'phoenix', employer_id: 1)
\ No newline at end of file
+Employee.create(name: 'Ryan Kwon', username: 'rhk1', password: 'phoenix', employer_id: 1)
+QrCode.create(title: "Stuff1", body: "Hey", company_id: "1")
```

Comment:
I recommend the `create!` method over `create` since the former will throw an exception.